t/t0024-content-s3.t

```diff
@@ -139,6 +163,17 @@ test_expect_success HAVE_JQ 'kvs-checkpoint.get foo returns same value' '
         test_cmp value2.exp value2.out
 '
 
+test_expect_success 'changing config file does not kill running backing store' '
+    err=0 && 
+    export tmp=$FLUX_CONF_DIR && export S3_CONF_DIR=/dev/null && \
+    flux config reload && \
+	for size in $SIZES; do \
+		if ! check_blob $size; then err=$(($err+1)); fi; \
+	done && \
+    export FLUX_CONF_DIR=$tmp && \
+	test $err -eq 0
+'
```

Comment:
This test shouldn't need to set FLUX_CONF_DIR, and S3_CONF_DIR isn't used any more.  Loading blobs is one way to check that the module didn't die, but maybe a better test would be that your message about the config reload not taking effect was emitted to the logs, e.g.

A suggested replacement test might be:
```sh
test_expect_success 'config: reload with good config logs notice of no effect' '
        flux config reload 2>/dev/null &&
        flux dmesg | grep content-s3 | tail -1 >reload.log &&
        grep "changes will not take effect until next flux restart" reload.log
'
```